main/settings/src/main/scala/sbt/Scope.scala

```diff
@@ -99,18 +99,69 @@ object Scope {
       case BuildRef(uri) => BuildRef(resolveBuild(current, uri))
     }
 
-  def display(config: ConfigKey): String = config.name + ":"
+  def display(config: ConfigKey): String = guessConfigIdent(config.name) + "/"
+
+  private[sbt] val configIdents: Map[String, String] =
+    Map(
+      "it" -> "IntegrationTest",
+      "scala-tool" -> "ScalaTool",
+      "plugin" -> "CompilerPlugin"
+    )
+  private[sbt] val configIdentsInverse: Map[String, String] =
+    configIdents map { _.swap }
```

Comment:
Nitpick: why `{}` for a simple operation that can be done with `()`? I think it's nicer the other way, could you write it with `()`?